### ISSUE TITLE
Don't test for payload header fields on HEAD

### ISSUE THREAD

**Issue by michielbdejong** [solid/test-suite#24]
According to https://tools.ietf.org/html/rfc7231#section-4.3.2 the following payload header fields MAY be omitted in a HEAD response:

|-------------------|----------------------------|
| Header Field Name | Defined in...              |
|-------------------|----------------------------|
| Content-Length    | Section 3.3.2 of [RFC7230] |
| Content-Range     | Section 4.2 of [RFC7233]   |
| Trailer           | Section 4.4 of [RFC7230]   |
| Transfer-Encoding | Section 3.3.1 of [RFC7230] |
|-------------------|----------------------------|

**Comment by kjetilk** [comment]
Right. But should we omit them? Content-Length may be important in planning what do with a response, to me it is one reason why you'd want to HEAD first to see if you are able to cope with the response...

**Comment by michielbdejong** [comment]
OK i can add Transfer-Encoding on HEAD, to get full points on this test.
But the http spec says 'MAY omit' and the solid spec doesn't overwrite that, so it's not really fair to substract points for this when servers do omit payload header fields.

**Comment by kjetilk** [comment]
Right, I can see that point. 

You know, there is the tension in my head between what we need to do in the future and the mays right now. 

I can certainly do this, but lets prioritize more tests first.
